logger/include/logger.hpp

```diff
@@ -45,4 +45,14 @@ void logMessage(
     const std::string& message,
     const std::source_location& location = std::source_location::current());
 
+/**
+ * @brief Fetches the latest logs from the buffer
```

Comment:
Re-phrase? It fetches number of loges as specified in the parameter.  re-word accordingly.

Reply:
done